Library/Homebrew/shims/super/make

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
 
+export MAKE=${MAKE:-make}
```

Comment:
Would it better to use `HOMEBREW_MAKE` as it is a private variable used by Homebrew?